Dockerfile.actions

```diff
@@ -0,0 +1,2 @@
+# commit_hash: 91e3b4b67ebfdd2709d531866ff68b15f76808dc
```

Comment:
This solution works in my private repository
![image](https://github.com/rerost/issue-creator/assets/5201588/d8e56851-6986-4221-8471-5aa11d6dc8a5)